src/components/sendV2/summary/summary.js

```diff
@@ -18,14 +18,16 @@ class Summary extends React.Component {
     this.state = {
       secondPassphrase: {
         hasSecondPassphrase: false,
-        isValid: '',
+        isValid: false,
         feedback: '',
+        value: null,
```

Comment:
shouldn't be `''`

Reply:
no based on how the transaction are submitted I tried like value: '' before but doenst work so this needs to be null, just this one